pxtsim/logs.ts

```diff
@@ -290,6 +300,18 @@ namespace pxsim.logs {
                 e.element.className = "ui log " + e.theme;
                 let raiseTrends = false;
                 if (e.accvalues) {
+                    e.accvaluesElement = document.createElement("span");
+                    e.accvaluesElement.className = "ui log " + e.theme + " gauge"
+                    e.chartElement = document.createElement("canvas");
+                    new CanvasChart().drawChart(e.chartElement, [new Point(e.accvalues[0].t, e.accvalues[0].v)])
```

Comment:
drawing chart should be deferred

Reply:
We should make the CanvasChart work with the .t, .v values instead of allocation "new Point" objects.

Reply:
Drawing chart actually _is_ deferred in serial.tsx, which is where I'm porting the functions from logs.ts.  My bad - logs.ts wasn't supposed to be in this PR.